DisguiseUnityRenderStream/Runtime/TemporaryTexture2DManager.cs

```diff
@@ -194,12 +192,10 @@ void DestroyTexture(Texture2D texture)
 #endif
         }
 
-        void DebugLog(bool condition, string message)
+        [Conditional("TEMPORARY_TEXTURE_2D_MANAGER_DEBUG")]
```

Comment:
Why do you need 2 different compiler defines when 1  will probably do? Or maybe even one of the built-in ones?

Reply:
I can unify under "LOG_TEXTURE_POOLING". Which built-in do you have in mind?

Reply:
I was trying to find something that indicates some kind of "debug" mode, but it looks like Unity only has the "development build" one that only applies to the player, so nothing comes to mind.